From 8d1420f3019cd1caccf2ffa15a5873f0c61ab529 Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppym@gentoo.org>
Date: Sun, 24 Jan 2021 22:39:03 -0500
Subject: [PATCH] tftpd: install into sbindir

The xinet.d config expects the daemon to live in sbindir.

Closes: https://github.com/iputils/iputils/pull/310

Reviewed-by: Petr Vorel <pvorel@suse.cz>
Signed-off-by: Mike Gilbert <floppym@gentoo.org>
---
 tftpd/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tftpd/meson.build b/tftpd/meson.build
index 6e508a24..b4cf6812 100644
--- a/tftpd/meson.build
+++ b/tftpd/meson.build
@@ -3,7 +3,8 @@ inc = include_directories('..')
 executable('tftpd', ['tftpd.c', 'tftpsubs.c', git_version_h],
 	include_directories : inc,
 	link_with : [libcommon],
-	install: true)
+	install: true,
+	install_dir: sbindir)
 
 subs = configuration_data()
 subs.set('sbindir', sbindir)
